assets/js/renderer.js

```diff
@@ -370,11 +370,11 @@ $(document).ready(function () {
             });
     }
 
-    // Open a SQLite Database File
-    openDBFileButton.click(function() {
+    // Open a SQLite Database File -- refactored to work with both button click and custom menu
```

Comment:
You can remove this comment. This type of comment is better for a commit message where you have the context of the change history.